lib/bokken/accounts/ninja.ex

```diff
@@ -26,6 +26,10 @@ defmodule Bokken.Accounts.Ninja do
 
     field :notes, :string
 
+    field :codemonkey, :string
+    field :lightbot, :string
+    field :codewars, :string
```

Comment:
Since a ninja is embedding many socials, you should add these to the `Social` schema. Follow what was made at #149 by me.

Reply:
I tried to follow what you said but I'm not getting something, shouldn't ninja's socials be separated from mentors' socials? I'm not understanding you pretend them together. :pray:

Reply:
They should be together. The responsibility of letting ninjas add/change their codemonkey, lightbot and codewars is from shuriken. Anyway, if you want, you can do a check when the transaction is made: if the user is a ninja, he can only change the mentioned socials (otherwise you should throw an error).